.github/workflows/ci.yaml

```diff
@@ -11,3 +11,8 @@ jobs:
         uses: actions/checkout@v1
       - name: Build & Test
         run: swift test -c release --enable-test-discovery
+      - name: Compile UseCases
+        run: |
+          cd UseCases
+          swift package resolve
```

Comment:
Not sure the explicit resolve is needed?

Reply:
`swift build` will do it automatically, but on CI I'd rather be explicit so that it's easier to find failures once they occur. And who knows, maybe we'll be able to even cache SPM dependencies 😁